fdbclient/SystemData.cpp

```diff
@@ -203,6 +203,9 @@ const KeyRangeRef writeConflictRangeKeysRange =
     KeyRangeRef(LiteralStringRef("\xff\xff/transaction/write_conflict_range/"),
                 LiteralStringRef("\xff\xff/transaction/write_conflict_range/\xff\xff"));
 
+const KeyRangeRef ddStatsRange = KeyRangeRef(LiteralStringRef("\xff\xff/metrics/data_distribution_stats/"),
```

Comment:
I think "\xff\xff/metrics/key_boundary_bytes" might be more descriptive, would love to get a second opinion though @sfc-gh-clin  @tclinken. It's similar to a [concept](https://apple.github.io/foundationdb/api-python.html#fdb.locality.fdb.locality.get_boundary_keys) that's already in the api

Reply:
I think the idea is that in the future there will be more stats added to these data distribution stats, e.g. read and write write, so we don't want to constrain the prefix to only being about the number of bytes in a shard.

Reply:
Can the values be json objects then?

Reply:
In this case, looks like prefix `\xff\xff/metrics/data_distribution_stats/key_boundary_bytes` for this feature and further read/write rate can use prefix `\xff\xff/metrics/data_distribution_stats/read_rate`. Lol, is this too long?

Reply:
When we expose another shard info (say read bandwidth) here, will the new info and the old info (i.e., shard size in this PR) be serialized in the returned key-value? 

How to avoid breaking the existing code that interpret the value as bytes?

Reply:
If we want to push this into 6.3, which I don't object, it's probably better to use a more specific name, say `\xff\xff/metrics/data_distribution/key_boundary_bytes`.

@sfc-gh-anoyes @tclinken @zjuLcg What do you think?

Reply:
Hi, meng. I just had a discussion with @sfc-gh-anoyes and @tclinken.
Since we plan to have more fields encoded in the future, we think it is better to make the value as a json string. Right now, it has only one filed like 
`{"ShardBytes": "<bytes>"}`